ios/Classes/FlutterSegmentPlugin.m

```diff
@@ -185,10 +185,18 @@ - (void)identify:(FlutterMethodCall*)call result:(FlutterResult)result {
     NSString *userId = call.arguments[@"userId"];
     NSDictionary *traits = call.arguments[@"traits"];
     NSDictionary *options = call.arguments[@"options"];
+      
+      if ([userId isEqual:[NSNull null]]){
```

Comment:
```suggestion
      if (!userId || [userId isEqual:[NSNull null]]){
```
Objc has 2 null's. One represents an object (NSNull) and one is just a regular (nil).

Reply:
Changed it a bit, not it better reflects what is the real problem.